src/osp/Active/SysAreaAssociate.cpp

```diff
@@ -131,9 +133,9 @@ void SysAreaAssociate::area_move(ActiveScene& rScene, Vector3s translate)
     auto &areaPosTraj = pArea->get_universe().get_reg()
             .get<universe::UCompTransformTraj>(pArea->m_areaSat);
 
-    areaPosTraj.m_position += translate;
+//    areaPosTraj.m_position += translate;
```

Comment:
commented out?

Reply:
This part is currently broken, I'll fix it in the next PR